core/src/main/java/bisq/core/grpc/CoreWalletsService.java

```diff
@@ -80,6 +80,15 @@ public long getAddressBalance(String addressString) {
         return btcWalletService.getBalanceForAddress(address).value;
     }
 
+    public String getAddressBalanceInfo(String addressString) {
+        var satoshiBalance = getAddressBalance(addressString);
+        var btcBalance = formatSatoshis.apply(satoshiBalance);
+        var numConfirmations = getNumConfirmationsForMostRecentTransaction(addressString);
+        return addressString
+                + "  balance: " + format("%13s", btcBalance)
+                + ((numConfirmations > 0) ? ("  confirmations: " + format("%6d", numConfirmations)) : "");
```

Comment:
A note: formatting logic in `getAddressInfo` and `getFundingAddresses` has a fair amount of duplication.

Reply:
I am going to take a look at this next, and any changes I make will be in a new PR associated with a  [5-getpaymentaccts](https://github.com/ghubstan/bisq/tree/5-getpaymentaccts) sub branch.